### ISSUE TITLE
how to run gebaca?

### ISSUE THREAD

**Issue by jason-s** [dmdmtg/gebaca#1]
So this brings back memories... I was a teenager visiting my father at his office at Bell Labs Holmdel in the late '80s, and we played this game a couple of times.

Is there an executable anywhere? How can I use a dmd5620 emulator to run this? I found a dmd5620 emulator https://apps.apple.com/us/app/dmd-5620/id1448142273?mt=12&uo=4&app=apps but not sure what to do next.

**Comment by DrDaveD** [comment]
Yes, binaries are available in the blit-binaries and mtg-binaries repositories here in https://github.com/dmdmtg.

As I recall, I got gebaca to execute on the emulator but it was unsatisfying because it ran too fast on a Mac, resulting in immediate death.  

Documentation about the emulator is available [here](https://loomcom.com/3b2/dmd5620_emulator.html).  In order to download programs, it needs to be connected to a host that is running layers and the [5620 development package](https://github.com/dmdmtg/dev5620) in order to use '32ld'.  I used the [3b2 emulator](https://loomcom.com/3b2/emulator.html) from the same person and 3b2 binary installs for layers and the development package.
